testing/tests/DevExpress.exporter/jspdfParts/jspdf.dataGrid.tests.js

```diff
@@ -30,51 +29,78 @@ const moduleConfig = {
     }
 };
 
-QUnit.module('Scenarios, generate autoTable options', moduleConfig, () => {
-    const getOptions = (context, dataGrid, expectedCustomizeCellArgs, options) => {
-        const { /* keepColumnWidths = true, selectedRowsOnly = false, */ autoTableOptions = {} } = options || {};
+QUnit.module('Scenarios, check autoTableOptions', moduleConfig, () => {
+    const getOptions = (context, dataGrid, options) => {
+        const { keepColumnWidths = true, /* selectedRowsOnly = false, */ autoTableOptions = {} } = options || {};
```

Comment:
You can remove redundant comment in the destructurization object.

Reply:
ok, removed